apps/yggdrasil/test/yggdrasil/character_test.exs

```diff
@@ -0,0 +1,156 @@
+defmodule Yggdrasil.CharacterTest do
```

Comment:
See your note with describe so with describe the macro could be 

```Elixir
describe "Caracter.changeset/2" do
  it "requires a valid user id" do

  end
  it "requires a valid game id" do

  end
end
```
That said when it's printed out I don't really care, practically having it repeat like this
```
"Caracter.changeset/2 requires a valid user id"
"Caracter.changeset/2 requires a valid game id"
```
makes it easier to understand the entire test you are looking without having to scroll up to the top and see what "it" is, but we may be able to change the test output as well.